addons/web/static/src/js/views/control_panel/control_panel_view.js

```diff
@@ -177,7 +177,7 @@ var ControlPanelView = Factory.extend({
                                 attrs.name ||
                                 attrs.domain ||
                                 'Ω';
-        if (attrs.invisible) {
+        if (JSON.parse(attrs.modifiers || '{}').invisible) {
```

Comment:
If I'm understanding the fix correctly: we only use the `modifiers` attribute keys and the `invisible` attribute is rendered pointless, is that right? If so shouldn't it be removed in this PR?

Reply:
@Arcasias `modifiers` are parsed server-side in ir.ui.view [`postprocess`](https://github.com/odoo/odoo/blob/13.0/odoo/addons/base/models/ir_ui_view.py#L898) and `invisible` attribute is kept in that process (in case the it need to be re-evaluated later on and is usefull for debugging [ex: from fields_view_get()] )

Reply:
@Arcasias Note: The regression windows is between 12.0 and saas-12.1

In v12.0 this is fine in [search_input.js `visible`](https://github.com/odoo/odoo/blob/12.0/addons/web/static/src/js/views/search/search_inputs.js#L83-L85), but in saas-12.1 it seems to be lost after https://github.com/odoo/odoo/commit/40dd121938552d3f88289d2e5fcd1d57a0dd01bb

Reply:
@Arcasias I've removed the "invisible" attributes on the test, let me know if there anything else to change. Thanks!